crates/zkevm_test_harness/src/witness/tracer/callstack_handler.rs

```diff
@@ -22,13 +22,16 @@ pub enum LogAction {
     ForwardNoRollback(usize),
 }
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct LogQueryWithAuxData {
+    pub marker: QueryMarker,
+    pub cycle: u32,
+    pub query: LogQuery,
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub enum ExtendedLogQuery {
-    Query {
-        marker: QueryMarker,
-        cycle: u32,
-        query: LogQuery,
-    },
+    Query(Box<LogQueryWithAuxData>),
```

Comment:
You can check for [`clippy::large_enum_variant`](https://rust-lang.github.io/rust-clippy/master/index.html#/large_enum_variant) lints to catch similar problem in other parts of the code.

Reply:
I'll check it after this PR. This particular case has a big impact, since we form large vectors of elements of this enum